Ronni-ronsbons/week-2/hw-js-arrays-functions-loops starter-code/main.js

```diff
@@ -0,0 +1,166 @@
+	(Part 1, Part 2, and Part 3) using a while-loop.
+*/
+i = 0;
+while (i < ages.length) {
+	console.log(ages[i]);
+	i++;
+};
+
+i = 0;
+while (ages[i] < 20) {
+	younger.push(ages[i]);
+	i++;
+};
+console.log(younger);
+
+i = younger.length - 1;
+while (i >= 0) {
+	console.log(younger[i]);
+	i--;
+};
+
+/*
+	Part 5: Create a function, getDayOfWeek, that will tell you what day of the week it is.
+	For example, getDayOfWeek(1) would return "Monday", and getDayOfWeek(5) would return "Friday".
+	An invalid day of the week should return the string, "Please enter a valid day of the week".
+
+	Hint: Using a "switch" statement might result in cleaner code
+*/
+
+// let getDayOfWeek = function (day) {
+// 	if (day === 0) {
+// 		console.log("Sunday");
+// 	} else if (day === 1) {
+// 		console.log("Monday");
+// 	} else if (day === 2) {
+// 		console.log("Tuesday");
+// 	} else if (day === 3) {
+// 		console.log("Wednesday");
+// 	} else if (day === 4) {
+// 		console.log("Thursday");
+// 	} else if (day === 5) {
+// 		console.log("Friday");
+// 	} else if (day === 6) {
+// 		console.log("Saturday");
+// 	} else if (day === 7) {
+// 		console.log("Sunday");
+// 	} else {
+// 		console.log("Please enter a valid day of the week");
+// 	};
+// };
+
+let getDayOfWeek = function (day) {
+	switch (day) {
+		case 0:
+			console.log("Sunday");
+			break;
+		case 1:
+			console.log("Monday");
+			break;
+		case 2:
+			console.log("Tuesday");
+			break;
+		case 3:
+			console.log("Wednesday");
+			break;
+		case 4:
+			console.log("Thursday");
+			break;
+		case 5:
+			console.log("Friday");
+			break;
+		case 6:
+			console.log("Saturday");
+			break;
+		case 7:
+			console.log("Sunday");
+			break;
+		default:
+			"Please enter a valid day of the week.";
+	};
+};
+
+getDayOfWeek(4);
+
+
+/*
+	Bonus - Part 6: All this work is making me hungry. Let's create a new function, addPepperoniToPizza,
+	that accepts an array of pizza ingredients. If the array of ingredients already contains
+	pepperoni, then return the pizza ingredients without doing anything to it. If the array of
+	ingredients does not contain pepperoni, then add pepperoni to the array and return it.
+
+	For example, addPepperoniToPizza(['onions', 'olives', 'pineapple']) should return
+	['onions', 'olives', 'pineapple', 'pepperoni'].
+
+	And addPepperoniToPizza(['spinach', 'pepperoni', 'mushrooms']) should return
+	['spinach', 'pepperoni', 'mushrooms'].
+
+	Hint: Take a look at the array.indexOf() method. How can we use this to check if we have pepperoni
+	in our array?
+	(https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/indexOf)
+*/
+
+let pizzaIngredients = ["olives", "sausage", "mushroom"];
+
+let addPepperoniToPizza = function () {
+	if (pizzaIngredients.indexOf("pepperoni") >= 0) {
+		console.log(pizzaIngredients);
+	} else {
+		pizzaIngredients.push("pepperoni");
+		console.log(pizzaIngredients);
+	};
+};
+
+addPepperoniToPizza();
+
+
```

Comment:
Looping looks solid - are you comfortable with these methods? There are more options available we'll go over later in the course.